apps/expo/src/components/EventTab.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Text, View } from "react-native";
+
+// TO-DO : Change to be Activity type when structure is ready
+export interface Event {
+  name: string;
+}
+
+interface EventProps {
+  activity: Event;
+  attending?: boolean;
+}
+
+export default function EventTab({ activity, attending }: EventProps) {
+  return (
+    <View className={`w-fill bg-p-95 flex-col justify-center rounded-md p-4`}>
+      <Text className="text-p-10 font-title-md">{activity.name}</Text>
+
+      <View className="flex-row items-center justify-start">
+        <Text className="text-n-40 font-body-md mr-3">Attendance:</Text>
```

Comment:
the textp-10 and text-n-40 shouldn't be necessary

Reply:
The font color is black otherwise 🤔

Reply:
<img width="322" alt="Screenshot 2024-02-10 at 9 52 39 PM" src="https://github.com/uoftblueprint/centre-for-dreams/assets/113125436/012c91e9-e855-40bd-859c-6d64f1045206">

Reply:
oh wait im dumb ur right